packages/build/tests/error/snapshots/tests.js.md

```diff
@@ -1502,25 +1502,9 @@ Generated by [AVA](https://ava.li).
       A Netlify Function failed to require one of its dependencies.␊
       Please make sure it is present in the site's top-level "package.json".␊
     ␊
-      In file "/file/path"␊
+      In file "packages/build/tests/error/fixtures/dependencies/functions/test.js"␊
     ␊
     ␊
-      Resolved config␊
-      build:␊
-        functions: /file/path␊
-    ␊
-    Error monitoring payload:␊
-    {␊
-      "errorClass": "dependencies",␊
-      "errorMessage": "A Netlify Function failed to require one of its dependencies./nPlease make sure it is present in the site's top-level /"package.json/"/file/path file /"/file/path"/file/path find module 'does-not-exist'/file/path stack:/n- /file/path",␊
-      "context": "Dependencies installation error",␊
-      "groupingHash": "Dependencies installation error/n",␊
-      "severity": "info",␊
-      "unhandled": false,␊
-      "pluginPackageJson": false,␊
-      "other": {␊
-        "groupingHash": "Dependencies installation error/n"␊
-      }␊
```

Comment:
Seems like we're missing the config resolution log here, is it expected?

Reply:
Indeed! I have a fix for this ready to commit.

Reply:
Fixed in https://github.com/netlify/build/pull/2456/commits/a88ab2fc336fcebd61107ed99c2ecc0a4303d7d9.